Aquarius spitting out nonstop ERRORs in ocean.py unit tests

**Summary**

If I run ocean.py unit tests (with barge), the log of the console running barge console gets filled with the following:
```console
aquarius_1         | 2021-03-16 14:13:27,523 - aquarius - ERROR - get_ddo: NotFoundError(404, '{"_index":"aquarius","_type":"_doc","_id":"did:op:8761f7322680D916fe081Acafb157B1ea5a1c743","found":false}')
aquarius_1         | 2021-03-16 14:13:27 2ce1d6ca440c root[50] ERROR Dao.get -- asset with id did:op:8761f7322680D916fe081Acafb157B1ea5a1c743 was not found, original error was:NotFoundError(404, '{"_index":"aquarius","_type":"_doc","_id":"did:op:8761f7322680D916fe081Acafb157B1ea5a1c743","found":false}')
aquarius_1         | 2021-03-16 14:13:27,740 - aquarius - ERROR - get_ddo: NotFoundError(404, 
...
```

**To reproduce**

Go through ocean.py's [developers.md](https://github.com/oceanprotocol/ocean.py/blob/main/READMEs/developers.md) up to the point of running unit tests. In the tests you will see the above issue.

To avoid running all the tests, even just running this will spit logs like the above:
```console
pytest ocean_lib/assets/test/test_ocean_assets.py
```

Just to add another data point to this issue, two of the `test_ocean_assets.py` tests fail.

```
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_register_asset - Exception: Unable to search for DDO: b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Ser...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_ocean_assets_search - Exception: Unable to search for DDO: b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Interna...
```

--- a/aquarius/app/assets.py
+++ b/aquarius/app/assets.py
@@ -3,6 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 import ecies
+import elasticsearch
 import json
 import logging
 import os
@@ -94,6 +95,8 @@ def get_ddo(did):
         return Response(
             sanitize_record(asset_record), 200, content_type="application/json"
         )
+    except elasticsearch.exceptions.NotFoundError:
+        return f"{did} asset DID is not in OceanDB", 404
     except Exception as e:
         logger.error(f"get_ddo: {str(e)}")
         return f"{did} asset DID is not in OceanDB", 404

--- a/aquarius/app/dao.py
+++ b/aquarius/app/dao.py
@@ -23,10 +23,8 @@ class Dao(object):
     def get(self, asset_id):
         try:
             asset = self.oceandb.read(asset_id)
-        except elasticsearch.exceptions.NotFoundError as e:
-            logging.error(
-                f"Dao.get -- asset with id {asset_id} was not found, original error was:{str(e)}"
-            )
+        except elasticsearch.exceptions.NotFoundError:
+            logging.info(f"Dao.get: asset with id {asset_id} was not found in OceanDB.")
             raise
 
         except Exception as e:

